index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Lato"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="css/main.css" />
+    <title>Microverse Concert</title>
+  </head>
+  <body>
+    <section class="navigation">
+      <nav class="top-bar">
+        <i class="fa fa-facebook"></i>
+        <i class="fa fa-instagram"></i>
+        <i class="fa fa-youtube"></i>
+        <a href="#">Login</a>
+      </nav>
+      <nav class="flex-nav">
+        <div class="menu-open">
+          <img src="assets/img/ic_menu.svg" alt="" />
+        </div>
+        <div class="menu">
+          <a href="index.html" class="logo"
+            ><img src="assets/img/logo.png" alt=""
+          /></a>
+          <a href="about.html">About</a>
+          <a href="#media">Media</a>
+          <a href="#lineup">Line up</a>
+          <a href="#buytickets">Buy tickets</a>
+          <div class="menu-close">
+            <img src="assets/img/Union.svg" alt="" />
+          </div>
+        </div>
+      </nav>
+    </section>
+    <header>
+      <p class="hello">"Hello Micronauts!"</p>
+      <h1 class="title">Microverse Concert Music for Developers 2021</h1>
+      <div class="intro">
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Accus antium
+          ipsam iste itaque nihil architecto, volu tatem sed facilis porro
+          maiores, rerum vitae.
+        </p>
+      </div>
+      <p class="date">05(FRI)-07(SUN) DEC 2021</p>
+      <p class="at">@ YouTube: Microverse Official Channel</p>
+    </header>
+    <main class="program">
+      <h2>Main Program</h2>
+      <div class="program-wrap">
+        <div class="program-item">
+          <img src="assets/img/icon1.png" alt="" />
+          <h3>Live Music</h3>
+          <p>
+            More than 50 artists amet consectetur adipisicing elit. Repudiandae
+            harum nisi, aliquid.
+          </p>
+        </div>
+        <div class="program-item">
+          <img src="assets/img/icon2.png" alt="" />
+          <h3>Visual arts</h3>
+          <p>
+            Trippy visuals from consectetur adipisicing elit. Repudiandae harum
+            nisi.
+          </p>
+        </div>
+        <div class="program-item">
+          <img src="assets/img/icon3.png" alt="" />
+          <h3>Interviews</h3>
+          <p>
+            On stage with amet consectetur. Repudiandae harum nisi, aliquid
+            adipisicing elit.
```

Comment:
- [ ] You have grouped the main program using a `main` tag, this shouldn't be so, as the `main` HTML element represents the dominant content of the `body` of a document. So instead of doing this, It's advisable to group the entire content of the page, ( except the header which contains the nav and the footer) using a `main` tag, and then the individual groups like `main program`, `features speakers` will be grouped using a `section` tag. 
Note that, while the usage of semantic HTML tags helps improve page structuring, reinforces the meaning of information in web pages, it is also important to be semantically correct when using them.
- [ ] Reduce the spacings between the articles in the main program.